yb-voyager/cmd/analyzeSchema.go

```diff
@@ -197,6 +198,30 @@ func reportSummary() {
 		reportStruct.Summary.DBObjects = append(reportStruct.Summary.DBObjects, dbObject)
 	}
 }
+// Checks Whether there is a GIN index
```

Comment:
In the comment, can you show examples (with full SQL DDL stmt) of all the scenarios reported by this function?